src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -356,8 +360,43 @@ public Result importVoDFolder(String vodFolderPath) {
 
 		return result;
 	}
+
+	private void sendUnexpectedShutdownHook() {
+		final String listenerHookURL = appSettings.getListenerHookURL();
+		if (listenerHookURL == null || listenerHookURL.isEmpty()) {
+			return;
+		}
+		logger.info("Setting timer to call unexpected server shutdown hook.");
+		vertx.setTimer(10, e -> notifyHook(listenerHookURL, null, HOOK_UNEXPECTED_SERVER_SHUTDOWN, null, null,
```

Comment:
Send server's hostname, address or similar things to identify the server in this hook